package/MDAnalysis/analysis/pca.py

```diff
@@ -121,11 +121,12 @@
 
 from MDAnalysis import Universe
 from MDAnalysis.analysis.align import _fit_to
-from MDAnalysis.lib.log import ProgressBar
+from MDAnalysis.lib.log import ProgressBar, ProgressBar_Dual
 
 from ..lib import util
 from ..due import due, Doi
 from .base import AnalysisBase
+from tqdm.auto import tqdm
```

Comment:
move imports of other packages above the MDA imports

Reply:
Done.